x-pack/plugins/security_solution/public/overview/components/overview_risky_host_links/index.tsx

```diff
@@ -7,18 +7,33 @@
 
 import React from 'react';
 
-import { GlobalTimeArgs } from '../../../common/containers/use_global_time';
-import { useRiskyHostLinks } from '../../containers/overview_risky_host_links/use_risky_host_links';
 import { RiskyHostsEnabledModule } from './risky_hosts_enabled_module';
 import { RiskyHostsDisabledModule } from './risky_hosts_disabled_module';
-export type RiskyHostLinksProps = Pick<GlobalTimeArgs, 'from' | 'to' | 'deleteQuery' | 'setQuery'>;
+import { useHostsRiskScore } from '../../containers/overview_risky_host_links/use_hosts_risk_score';
+import { HostsRiskScore } from '../../../../common';
+import { LinkPanelListItem } from '../link_panel';
+export interface RiskyHostLinksProps {
+  timerange: { to: string; from: string };
+}
 
-const RiskyHostLinksComponent: React.FC<RiskyHostLinksProps> = (props) => {
-  const { listItems, isModuleEnabled } = useRiskyHostLinks(props);
+const getListItemsFromHits = (items: HostsRiskScore[]): LinkPanelListItem[] => {
```

Comment:
this logic seems to belong to RiskyHostsEnabledModule component to me with the updated structure

Reply:
Moving it! 👍